File: member-service/src/main/java/com/hedvig/memberservice/web/dto/UpdateContactInformationRequest.java
package com.hedvig.memberservice.web.dto;

import lombok.Data;
import org.springframework.lang.Nullable;

import java.time.LocalDate;

@Data
public class UpdateContactInformationRequest {

  private String memberId;
  private String firstName;
  private String lastName;
  private String email;
  private String phoneNumber;
  private Address address;
  @Nullable
  private LocalDate birthDate;
}

File: member-service/src/main/java/com/hedvig/memberservice/web/dto/UpdateContactInformationRequest.kt
package com.hedvig.memberservice.web.dto

import java.time.LocalDate

class UpdateContactInformationRequest(
    val memberId: String,
    val firstName: String? = null,
    val lastName: String? = null,
    val email: String? = null,
    val phoneNumber: String? = null,
    val address: Address? = null,
    val birthDate: LocalDate? = null
)
